File: LatB/Python/Cpu.py
class Cpu():
    # Constructor untuk class Cpu
    def __init__(self):
        # Attribute-attribute untuk class Cpu
        self.nama = ""
        self.harga = 0
    # Get dan Set tiap attribute
    def setNama(self, nama):
        self.nama = nama
    def getNama(self):
        return self.nama
    def setHarga(self, harga):
        self.harga = harga
    def getHarga(self):
        return self.harga
    # Fungsi untuk menampilkan isi attribute dari class Cpu
    def printCpu(self):
        print("CPU Name: " + str(self.nama))
        print("CPU Price: " + str(self.harga))

File: LatB/Java/Cpu.java
public class Cpu{
    // Attribute class Cpu
    private String nama;
    private int harga;
    // Constructor kosong
    Cpu(){}
    // Constructor
    Cpu(String nama, int harga){
        this.nama = nama;
        this.harga = harga;
    }
    // Get dan Set
    public String getNama() {
        return nama;
    }
    public void setNama(String nama) {
        this.nama = nama;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }
    // Fungsi untuk menampilkan isi dari attribute
    public void printCpu(){
        System.out.println("CPU Name: " + this.nama);
        System.out.println("CPU Price: " + this.harga);
    }
}